src/components/Logo/Logo.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import Svg from './style';
+
+const Logo = () => (
```

Comment:
First of all, let's create `Icons` folder inside of components and move this logo icon here. Then let's add ability to pass some props to this icon, e.g. `fill`, `width`, `height` and maybe `viewBox` (but also don't forget about default props)